### ISSUE TITLE
Linting (Github Actions)

### ISSUE THREAD

**Issue by ZenaMel** [Zenahr/lazystarter#22]
We should (in the near future) setup CI with linting support.
Example: https://github.com/fredrikaverpil/mylittleci

**Comment by ZenaMel** [comment]
flake8 (?)

**Comment by y4shb** [comment]
@Zenahr I'd like to give this a try. I've worked with CI before so I can write a config.yml file? And use pyment to implement flake8 formatting?

**Comment by ZenaMel** [comment]
Yeah I'd love that. Feel free to hack away. We can always tweak things later on. And yeah either flake8 or black is fine imo.

**Comment by y4shb** [comment]
Cool. I'm on it already. Can you assign me?
